Makefile

```diff
@@ -33,9 +34,20 @@ GOPROXY := https://proxy.golang.org
 endif
 export GOPROXY
 
+# Use GOPRIVATE environment variable if set
+GOPRIVATE := $(shell go env GOPRIVATE)
```

Comment:
What's the benefit of doing this vs just referencing GOPRIVATE directly, or using `GOPRIVATE ?=GOPRIVATE`?

Reply:
It seems to better use common approach, to avoid confusion, but probably `GOPRIVATE ?=GOPRIVATE` should work also.
Fox example, for it's not working as expected for `GOMODCACHE` default values, i.e.
```bash
[xyz@fedora ~]$ go env GOMODCACHE
/home/xyz/go/pkg/mod
[xyz@fedora ~]$ echo $GOMODCACHE

```